backend/src/main/kotlin/hu/bme/sch/cmsch/component/sheets/SheetsSetupWizard.kt

```diff
@@ -0,0 +1,287 @@
+    title = "Integráció készítése",
+    description = "",
+    wide = false,
+    adminMenuService = adminMenuService,
+    component = sheetsComponent,
+    auditLog = auditLogService,
+    showPermission = StaffPermissions.PERMISSION_CREATE_SHEETS,
+    adminMenuCategory = null,
+    adminMenuIcon = "add_circle",
+    adminMenuPriority = 2
+) {
+
+    private final val objectMapper = jacksonObjectMapper()
+    private final val formStructReader: ObjectReader = objectMapper.readerFor(object : TypeReference<List<FormElement>>() {})
+
+    private val permissionCard = DashboardPermissionCard(
+        1,
+        permission = showPermission.permissionString,
+        description = "Ez a jog szükséges ennek az oldalnak az olvasásához.",
+        wide = wide
+    )
+
+    override fun getComponents(user: CmschUser, requestParams: Map<String, String>): List<DashboardComponent> {
+        return listOf(
+            permissionCard,
+            DashboardDocsCard(
+                id = 2,
+                title = "Hogyan lehet?",
+                description = "",
+                wide = wide,
+                markdownContent = "Ahhoz, hogy integrációt hozz létre: menj bele pl. az úrlapok menübe és kattints az integráció hozzáadása ikonra!"
+            )
+        )
+    }
+
+    @GetMapping("/form/{formId}")
+    fun viewForm(model: Model, auth: Authentication, @RequestParam requestParams: Map<String, String>, @PathVariable formId: Int): String {
+        val user = auth.getUser()
+        adminMenuService.addPartsForMenu(user, model)
+        if (showPermission.validate(user).not()) {
+            model.addAttribute("permission", showPermission.permissionString)
+            model.addAttribute("user", user)
+            auditLog.admin403(user, component.component, "GET /${view}", showPermission.permissionString)
+            return "admin403"
+        }
+
+        model.addAttribute("title", title)
+        model.addAttribute("description", description)
+        model.addAttribute("view", view)
+        model.addAttribute("wide", wide)
+        model.addAttribute("components", getComponentsForForms(user, requestParams, formId))
+        model.addAttribute("user", user)
+        model.addAttribute("card", requestParams.getOrDefault("card", "-1").toIntOrNull())
+        model.addAttribute("message", requestParams.getOrDefault("message", ""))
+
+        return "dashboard"
+    }
+
+    fun getComponentsForForms(user: CmschUser, requestParams: Map<String, String>, formId: Int): List<DashboardComponent> {
+        val token = requestParams.getOrDefault("token", UUID.randomUUID().toString())
+        val name = requestParams.getOrDefault("name", "névtelen integráció")
+        return listOf(
+            permissionCard,
+            DashboardDocsCard(
+                id = 2,
+                title = "Űrlap és Google sheets összekötése",
+                description = "",
+                wide = wide,
+                markdownContent = """
```

Comment:
QoL: Add copy button option to card contents

Reply:
good idea